suite-native/atoms/src/NumPadButton.tsx

```diff
@@ -31,7 +33,7 @@ export const NumPadButton = ({ value, onPress, style, ...props }: NumPadButtonPr
             underlayColor={utils.colors.backgroundTertiaryPressedOnElevation0}
             {...props}
         >
-            <></>
+            <Text>{'\u00B7'}</Text>
```

Comment:
nit: shoudn't we rather use https://www.compart.com/en/unicode/U+2022 ?

Reply:
Yes, good point. Fixed